rsmtool/utils.py

```diff
@@ -429,7 +429,7 @@ def covariance_to_correlation(m):
     if not numrows == numcols:
         raise ValueError('Input matrix must be square')
 
-    Is = np.sqrt(np.abs(1 / np.diag(m)))
+    Is = np.sqrt(1 / np.diag(m))
```

Comment:
Why are we removing `np.abs` here?

Reply:
We shouldn't need `np.abs()` anymore, and it shouldn't exist (if we're mirroring the `poor()` function). 

The reason we were getting negative numbers here was because some of the covariance matrices that we were inverting had determinants  _extremely_ small (and sometimes negative), but not quite zero. So, we weren't getting a `LinAlgError` errors from `inv()`, but we were getting ill-conditioned matrices. 

Using `pinv()` in these cases should resolve the problem, and we shouldn't get negative elements in the diagonal even after inversion.